addons/pos_restaurant/static/src/app/floor_screen/floor_screen.js

```diff
@@ -385,6 +385,8 @@ export class FloorScreen extends Component {
         }
         const orders = this.pos.getTableOrders(table.id);
```

Comment:
looks like you also need to put the codes from 386 to 394 inside the `else` of the `if` statement just above. That way, when the `"Multiple open orders"` popup is shown and the user cancelled, we don't open the table.

Reply:
Hello @caburj 
Changes are done.
Can you please check again?